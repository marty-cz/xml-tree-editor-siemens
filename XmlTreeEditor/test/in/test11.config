<config>
  <input>test11.xml</input>
</config>