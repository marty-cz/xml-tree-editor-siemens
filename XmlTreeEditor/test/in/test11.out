Error: Input xml file "test03.xml" is invalid.